docs/website/docs/walkthroughs/deploy-a-pipeline/deploy-with-modal.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Deploy with Modal
+description: How to deploy a pipeline with Modal
+keywords: [how to, deploy a pipeline, Modal]
+canonical: https://modal.com/blog/analytics-stack
+---
+
+# Deploy with Modal
+
+## Introduction to Modal
+
+[Modal](https://modal.com/) is a serverless platform designed for developers. It allows you to run and deploy code in the cloud without managing infrastructure.
+
+With Modal, you can perform tasks like running generative models, large-scale batch jobs, and job queues, all while easily scaling compute resources.
+
+### Modal features
+
+- Serverless Compute: No infrastructure management; scales automatically from zero to thousands of CPUs/GPUs.
+- Cloud Functions: Run Python code in the cloud instantly and scale horizontally.
+- GPU/CPU Scaling: Easily attach GPUs for heavy tasks like AI model training with a single line of code.
+- Web Endpoints: Expose any function as an HTTPS API endpoint quickly.
+- Scheduled Jobs: Convert Python functions into scheduled tasks effortlessly.
+
+To know more, please refer to [Modals's documentation.](https://modal.com/docs)
+
+## Building data pipelines with dlt
+
+dlt is an open-source Python library that allows you to declaratively load data sources into well-structured tables or datasets. It does this through automatic schema inference and evolution. The library simplifies building data pipelines by providing functionality to support the entire extract and load process.
+
+### How does dlt integrate with Modal for pipeline orchestration?
+
+As an example of how to set up a pipeline in Modal, we'll use the [building a cost-effective analytics stack with Modal, dlt, and dbt.](https://modal.com/blog/analytics-stack) case study.
+
+The example demonstrates automating a workflow to load data from Postgres to Snowflake using `dlt`.
+
+## How to run dlt on Modal
+
+Here’s our `dlt` setup copying data from our Postgres read replica into Snowflake:
+
+1. Run the `dlt` SQL database setup to initialize their `sql_database_pipeline.py` template:
```

Comment:
```suggestion
1. Run the `dlt` CLI command to  initialize the SQL database source and setup the `sql_database_pipeline.py` template:
```

Reply:
updated

Reply:
Thank you! However, I don't see these changes on GitHub. Is there a chance you haven't pushed the updates to GitHub?